src/plots/stackedBarPlot.ts

```diff
@@ -148,7 +148,7 @@ export class StackedBar<X, Y> extends Bar<X, Y> {
       if (maximum.extent !== baselineValue) {
         // only draw sums for values not at the baseline
 
-        const text = this.labelFormatter()(maximum.extent);
+        const text = this.labelFormatter()(maximum.extent, maximum.axisValue, maximum.datasets);
```

Comment:
similarly - is there a way to get the specific dataset associated with this datum now?